### PR TITLE
CCD-3885 whitelist nfdiv backend and frontend

### PR DESCRIPTION
**Pull request by marvincudjoe**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CCD-3885

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by hemantt — APPROVED** [review]

**Review by aktaskaan — APPROVED** [review]
service ids need to be added to **CASE_DOCUMENT_S2S_AUTHORISED_SERVICES** env variable in flux per requested environment.

File: src/main/resources/service_config.json [hunk 1]
```diff
@@ -273,5 +273,27 @@
         "READ",
         "UPDATE"
       ]
-    }]
+    },
+      {
+        "id": "nfdiv_case_api",
```
Comment (rajaram-nagarajan) [inline src/main/resources/service_config.json:278]: @marvincudjoe - add the service names nfdiv_case_api and divorce_frontend in CASE_DOCUMENT_S2S_AUTHORISED_SERVICES for non-prod env.

File: src/main/resources/service_config.json [hunk 2]
```diff
@@ -273,5 +273,27 @@
...
+        "caseTypeId": ["NFD", "NO_FAULT_DIVORCE_BulkAction"],
+        "jurisdictionId": "DIVORCE",
+        "permissions": [
+          "CREATE",
+          "READ",
+          "UPDATE",
+          "DELETE"
```
Comment (rajaram-nagarajan) [inline src/main/resources/service_config.json:285]: Remove DELETE if it is not required by the service team.

File: src/main/resources/service_config.json [hunk 3]
```diff
@@ -273,5 +273,27 @@
         "READ",
         "UPDATE"
       ]
-    }]
+    },
+      {
```
Comment (rajaram-nagarajan) [inline src/main/resources/service_config.json:277]: (minor) alignment correction needed

File: src/main/resources/service_config.json (hunk 2 above)
Comment (rajaram-nagarajan) [reply src/main/resources/service_config.json:285]: service team confirmed - good to keep delete permission

File: src/main/resources/service_config.json (hunk 3 above)
Comment (marvincudjoe) [reply src/main/resources/service_config.json]: Done

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

**Pull request merged**